tests/replicaset-failover/mongoinsertbatch-execute-002.phpt

```diff
@@ -52,7 +52,7 @@ echo "Test completed\n";
 Killing master after 100
 Killing done
 Restarting done
-MongoCursorException: localhost:30200: Remote server has closed the connection
+MongoCursorException: localhost:%d: Remote server has closed the connection
```

Comment:
@derickr and I confirmed that yesterday. It certainly is odd.

Perhaps we should hold off on merging this for a bit.